master/usage/reduce-nodes.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Scheduling to well-known nodes
+canonical_url: 'https://docs.projectcalico.org/master/usage/reduce-nodes'
+---
+
+If you are on Kubernetes, use the Kubernetes API datastore, and have a cluster of more
+than 50 nodes, you have probably deployed Typha. Typha is a fan-out proxy that improves
+performance in larger clusters. Typha agents must accept connections from other agents on
```

Comment:
I think it would be helpful to mention that Typha uses host networking. This helps the reader understand why they need to open the port, since a service using pod networking does not need ports exposed on the host.  For example:

"Typha is part of {{site.prodname}} bootstrap infrastructure, and must be available before pods can be networked, so it uses host networking and opens a port on the node it is scheduled on."

Reply:
OK nice, I like this... 💯